### PR TITLE
EXLM 19 Tabbed Card Block

### PR DESCRIPTION
**Pull request by devisree29**
**Merge this PR after PR 299.**

Tabbed Catd Block 

Jira ID:https://jira.corp.adobe.com/browse/EXLM-19

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/n/tabbedtest
- After: https://feature-exlm-19-tabbedcards--exlm--adobe-experience-league.hlx.live/en/tabbedtest
- https://main--exlm--devisree29.hlx.live/en/tabbed
Tabbed Block Card
image

Desktop Performace
image
Mobile Performance
image

### CODE REVIEW

File: blocks/tabbed-cards/tabbed-cards.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      tabLabelsListElements.forEach((label) => {
+        label.classList.remove('active');
+      });
+      // Clear existing cards
+      const tabbedContent = blockElement.querySelector('.tabbed-cards-block');
+      tabLabel.classList.add('active');
+      if (tabbedContent) {
+        [...tabbedContent.children].forEach((cards) => {
+          cards.remove();
+        });
```
Comment (noormd08) [inline blocks/tabbed-cards/tabbed-cards.js:119]: Add innerHTML with empty value to clear the container.

File: blocks/tabbed-cards/tabbed-cards.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  // Update view link for initial content type
+  viewLinkURLElement.innerHTML = placeholders[`viewAll${convertToTitleCase(viewLinkInitialMappingKey)}`];
+  viewLinkURLElement.setAttribute('href', placeholders[`viewAll${convertToTitleCase(viewLinkInitialMappingKey)}Link`]);
+  tabList.appendChild(viewLinkURLElement);
+  tabList.children[0].children[0].classList.add('active');
+
+  // Render Block content
+  fetchDataAndRenderBlock(initialContentType, blockElement);
+  decorateIcons(headerDiv);
+}
```
Comment (noormd08) [inline blocks/tabbed-cards/tabbed-cards.js:153]: Add / Remove shimmer on every tab click

File: blocks/tabbed-cards/tabbed-cards-constants.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    MAPPING_KEY: 'community',
+    LABEL: 'Community',
+  },
+  CERTIFICATION: {
+    MAPPING_KEY: 'certification',
+    LABEL: 'Certification',
+  },
+});
+
+export default CONTENT_TYPES;
```
Comment (noormd08) [inline blocks/tabbed-cards/tabbed-cards-constants.js:33]: As discussed with Sivaram, we will keep the constants only for browse cards now. Please use the placeholders for these labels.

**Review by Nehaadobe — APPROVED** [review]

File: blocks/tabbed-cards/tabbed-cards-constants.js (hunk 3 above)
Comment (devisree29) [reply blocks/tabbed-cards/tabbed-cards-constants.js]: Removed the constants file

**Review by noormd08 — APPROVED** [review]

**Review by ruhisingh1 — APPROVED** [review]

**Pull request merged**